integrations/bitdefender/mappers/to-company.ts

```diff
@@ -0,0 +1,18 @@
+import type { BitdefenderCompany } from '../../models';
+
+export function toCompany(result: any): BitdefenderCompany {
+    return {
+        id: result.id,
+        name: result.name,
+        type: result.type,
+        country: result.country,
+        createdAt: new Date().toISOString(), // API doesn't return creation date, using current date
```

Comment:
remove createdAt if it isn't returned from the API. Be sure to remove from the yaml as well